neo4j-store/src/main/kotlin/org/ostelco/prime/storage/graph/Neo4jStore.kt

```diff
@@ -1133,24 +1134,39 @@ object Neo4jStoreSingleton : GraphStore {
                 }
     }
 
+    private val appNotifier by lazy { getResource<AppNotifier>() }
+
     override fun updateScanInformation(scanInformation: ScanInformation, vendorData: MultivaluedMap<String, String>): Either<StoreError, Unit> = writeTransaction {
         logger.info("updateScanInformation : ${scanInformation.scanId} status: ${scanInformation.status}")
         getCustomerUsingScanId(scanInformation.scanId, transaction).flatMap { customer ->
             scanInformationStore.update(scanInformation, transaction).flatMap {
                 logger.info("updating scan Information for : ${customer.contactEmail} id: ${scanInformation.scanId} status: ${scanInformation.status}")
-
+                val extendedStatus = scanInformationDatastore.getExtendedStatusInformation(vendorData)
                 if (scanInformation.status == ScanStatus.APPROVED) {
 
                     logger.info("Inserting scan Information to cloud storage : id: ${scanInformation.scanId} countryCode: ${scanInformation.countryCode}")
                     scanInformationDatastore.upsertVendorScanInformation(customer.id, scanInformation.countryCode, vendorData)
                             .flatMap {
+                                appNotifier.notify(
+                                        customerId = customer.id,
+                                        title = "eKYC Status",
```

Comment:
Should we move to constant for the common strings?

https://blog.egorand.me/where-do-i-put-my-constants-in-kotlin/

Reply:
We are implementing consts in a different way